lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,10 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Updating direction of scroll by calculating pixels.If the value is 0,
+		// the gradebook should be moved all the way to the left side of the screen.
+		// maxScroll is difference between table width and its parent width.
```

Comment:
Hey @singingwolfboy, @amir-qayyum-khan and @pdpinch have given me some context, and asked if I would flesh out the comments a bit, to clarify intended functionality. I'll push a commit shortly, and would welcome your feedback on whether my changes are helpful.